tests/testsuite/cfg.rs

```diff
@@ -543,12 +543,12 @@ fn cfg_raw_idents() {
         .build();
 
     p.cargo("check")
-        .with_status(101)
         .with_stderr_data(str![[r#"
-[ERROR] failed to parse manifest at `[ROOT]/foo/Cargo.toml`
-
-Caused by:
-  failed to parse `any(r#fn, r#all, r#target_os = "<>")` as a cfg expression: unexpected character `#` in cfg, expected parens, a comma, an identifier, or a string
+[WARNING] [[ROOT]/foo/Cargo.toml] future-incompatibility: `cfg(r#fn)` is deprecated as `r#fn` is a keyword and not an identifier and should not have have been accepted in this position.
```

Comment:
Are raw identifiers like this meant to warn?

Reply:
nm future commit handles it

Reply:
No, certainly not, it's the point of raw-ident to well be ident and not keywords, it's a bug, I will fix it.